README.rst

```diff
@@ -86,12 +86,13 @@ The default output is plain text of maximum width 79 characters.
 This can be adjusted using the ``--width`` parameter. To disable
 shortening altogether use ``--width=0``. The default width value
 can be saved in the config file as well. Use ``--format=wiki`` to
-enable simple MoinMoin wiki syntax. For stats which support them,
+enable simple MoinMoin wiki syntax or ``--format=wiki`` to enable
+markdown syntax. For stats which support them,
 ``--brief`` and ``--verbose`` can be used to specify a different
 level of detail to be shown.
 
---format=FMT
-    Output style, possible values: text (default) or wiki
+--format {text,md,wiki}
```

Comment:
```suggestion
--format {text|md|wiki}
```

Perhaps, `|` is more commonly used to separate alternative values?

Reply:
This is the separator used by `argparse` in the `--help` output. I've no problem with your suggestion for the README if you're ok it won't match the help output. Let me know what you prefer.